language-swift_5cd0480e2c7d3a177d6e6388.md

```diff
@@ -1,42 +1,47 @@
-Semaphore supports building, testing and releasing Swift applications in fully
```

Comment:
People discover this page either by browsing the list of supported languages or by typing 'swift' in the search box. For consistency we want to say the name of the language on every language page.

That being said, it looks like everything being said on the page, including the hello world YAML, is perfectly valid for both Swift and Objective-C projects (i.e. any Xcode based project)?